uber-lyft.ipynb

```diff
@@ -30,9 +30,8 @@
     "\n",
     "cluster = coiled.Cluster(\n",
     "    n_workers=30,\n",
-    "    account=\"events\",\n",
     "    name=\"uber-lyft\",\n",
-    "    shutdown_on_close=False,\n",
```

Comment:
Okay, now I'm beginning to think this is intentional. Maybe this was used to keep cluster hot for demos at the booth...

Reply:
Correct